todo_app/templates/index.html

```diff
@@ -35,40 +35,50 @@ <h2>Items</h2>
       <!-- List the todo items here -->
       <div class="collumn">
         <h2>Not Started</h2>
-        {% for to_do in lists.values()%}
-        {% if to_do.status == 'Not Started'%}
+        {% for to_do in lists.to_do_items%}
         <ul class="list-group mb-4">
           <li>
             <h3><a href="/to_do/{{to_do.id}}"> {{to_do.name}}</a></h3>
           </li>
         </ul>
-        {% endif %}
         {% endfor %}
       </div>
       <div class="collumn">
         <h2>Started</h2>
-        {% for to_do in lists.values()%}
-        {% if to_do.status == 'Started'%}
+        {% for to_do in lists.doing_items%}
         <ul class="list-group mb-4">
           <li>
             <h3><a href="/to_do/{{to_do.id}}"> {{to_do.name}}</a></h3>
           </li>
         </ul>
-        {% endif %}
         {% endfor %}
       </div>
 
       <div class="collumn">
         <h2>Done</h2>
-        {% for to_do in lists.values()%}
-        {% if to_do.status == 'Done'%}
-        <ul class="list-group mb-4">
-          <li>
-            <h3><a href="/to_do/{{to_do.id}}"> {{to_do.name}}</a></h3>
-          </li>
-        </ul>
+        {% if lists.done_items|length < 5 %}
+          {% for to_do in lists.done_items%}
+          <ul class="list-group mb-4">
+            <li>
+              <h3><a href="/to_do/{{to_do.id}}"> {{to_do.name}}</a></h3>
+            </li>
+          </ul>
+          {% endfor %}
+        {%endif%}
+
+        {% if lists.done_items|length >= 5 %}
```

Comment:
This works, but it leaves the logic in our HTML template which can be harder to maintain; a nicer solution might be to add a new property to the ViewModel that determines this for us and provides a friendly name, e.g.:
```suggestion
        {% if not lists.should_show_all_done_items() %}
```
(There is probably a still clearer name out there!)